src/main/idea/spotify-checkstyle-idea.xml

```diff
@@ -41,13 +55,18 @@
     <option name="BINARY_OPERATION_SIGN_ON_NEXT_LINE" value="true" />
     <option name="TERNARY_OPERATION_WRAP" value="1" />
     <option name="TERNARY_OPERATION_SIGNS_ON_NEXT_LINE" value="true" />
+    <option name="KEEP_SIMPLE_METHODS_IN_ONE_LINE" value="true" />
```

Comment:
just curious does this mean display a "simple method" as one line, or literally don't have line breaks after `{` and before `}`?